pkg/config/config.go

```diff
@@ -64,8 +64,14 @@ type Config struct {
 
 // TODO: https://github.com/jbrekelmans/kube-compose/issues/64
 // nolint
-func New() (*Config, error) {
-	fileName := "docker-compose.yml"
+func New(file string) (*Config, error) {
+	fileName := "docker-compose.yaml"
+	if File != "" {
```

Comment:
`File` needs to be lower case...

Reply:
As discussed `-f ""` on the command line will make it load the default file, this is different from docker-compose behavior.